### ISSUE TITLE
Rasa x

### ISSUE THREAD

**Issue by robertson-oa** [RasaHQ/rasa#11525]
### Rasa Open Source version

rasa==2.8.0

### Rasa SDK version

Rasa SDK Version==2.8.6

### Python version

3.8

### What operating system are you using?

Linux

### What happened?

Getting this error when running rasa x aftyer installation

### Command / Request

```shell
rasa x
```

### Relevant log output

```shell
ImportError: cannot import name 'local' from 'rasax.community'
```

**Comment by SumitVerma11** [comment]
I am also facing the same issue, does anyone know the solution?

**Comment by robertson-oa** [comment]
No solution given yet

**Comment by amn41** [comment]
do you have rasa x installed? (running `rasa --version` should tell you). Important to note though that we are [no longer maintaining Rasa X](https://rasa.com/blog/rasa-x-community-edition-changes/) .

**Comment by robertson-oa** [comment]
Yes I have rasa x installed
![Screenshot from 2022-09-20 15-57-51](https://user-images.githubusercontent.com/77049910/191307003-e6e91e21-b180-490f-9094-12cb1a2b296a.png)

**Comment by robertson-oa** [comment]
![Screenshot from 2022-09-20 15-57-51](https://user-images.githubusercontent.com/77049910/191307211-bda01398-afa5-4175-9d22-fb39a099112a.png)

**Comment by amn41** [comment]
the most recent rasa x version which is compatible with Rasa Open Source 2.8 is rasa x 1.0.4 (1.1 and later are only compatible with Rasa Open Source 3+).

**Comment by robertson-oa** [comment]
Okay so how do I install rasa x with the specific version

**Comment by tmbo** [comment]
@robertson-oa mhm which way did you use to install rasa x?

**Comment by robertson-oa** [comment]
@tmbo From the rasa X documentation: pip3 install rasa-x --extra-index-url https://pypi.rasa.com/simple

But I want to be able to install specific version 1.0.4

**Comment by tmbo** [comment]
Right, you should be able to do that using `pip3 install rasa-x==1.0.4 --extra-index-url https://pypi.rasa.com/simple`

**Comment by tmbo** [comment]
similarly, @SumitVerma11 please either install a version that corresponds to your installed Rasa Open Source version https://rasa.com/docs/rasa-enterprise/changelog/compatibility-matrix/.

TL;DR:
If you are using Rasa Open Source 2.x: `pip3 install rasa-x==1.0.4 --extra-index-url https://pypi.rasa.com/simple`
If you are using Rasa Open Source 3.x: `pip3 install rasa-x==1.1.3 --extra-index-url https://pypi.rasa.com/simple`

**Comment by robertson-oa** [comment]
@tmbo It worked thanks for your feedback
